robottelo/upgrade_utility.py

```diff
@@ -125,3 +125,17 @@ def publish_content_view(org=None, repolist=None):
     call_entity_method_with_timeout(content_view.publish, timeout=3400)
     content_view = content_view.read()
     return content_view
+
+
+def dependent_scenario_name(request_obj):
```

Comment:
This could actually be implemented as a fixture, not a function that has to be called within the test.

Not necessary for this PR, but something to consider doing for all these upgrade tests that depend on this pattern.

Reply:
yes, you are right @mshriver, I have created a common upgrade fixtures file and added it there.